src/sentry/message_filters.py

```diff
@@ -235,6 +235,8 @@ def _localhost_filter(project_config, data):
         # Dragon Web Extension from Nuance Communications
         # See: https://forum.sentry.io/t/error-in-raven-js-plugin-setsuspendstate/481/
         'plugin.setSuspendState is not a function',
+        # lastpass
+        'should_do_lastpass_here'
```

Comment:
Add a trailing comma plz.

Reply:
Shoot! Meant to. Will do. Thanks for catching that.